utilities/rali/rali_unittests/rali_unittests.cpp

```diff
@@ -160,7 +160,7 @@ int test(int test_case, int reader_type, int pipeline_type, const char *path, co
                 std::cout << "\n json_path has to be set in rali_unit test manually";
                 exit(0);
             }
-            meta_data = raliCreateCOCOReader(handle, json_path, true);
+            meta_data = raliCreateCOCOReader(handle, json_path, true, true);
```

Comment:
Please have a mask variable to pass for either segmentation / detection instead of hard-coding to true

Reply:
Resolved in the latest commit. I have added a seperate pipeline type for the user to choose and run the coco segmentation example in the unittests. I also added a test case for resize_mirror_normalize augmentation as well.